wiso.js

```diff
@@ -2,14 +2,14 @@
 	"translatorID": "136d5c30-d8b1-476f-9564-702a41b6126e",
 	"label": "wiso",
 	"creator": "Philipp Zumstein",
-	"target": "^https?://www\\.wiso-net\\.de/webcgi\\?",
+	"target": "^https?://www\\.wiso-net\\.de/(webcgi\\?|document/|dosearch\\?)",
```

Comment:
There are no tests that cover `webcgi` or `dosearch` parts. Either add tests (unless it's not possible) or drop them from here.